services/QuillLMS/app/views/pages/comprehension.html.erb

```diff
@@ -1 +1 @@
-<%= react_component('App', props: {})%>
+<%= react_component('App', props: { user: current_user.name&.split(" ")[0] })%>
```

Comment:
I think this is going to error if there is no current user?

Reply:
good catch-- I'll update